moped-editor/src/utils/mapHelpers.js

```diff
@@ -756,61 +728,46 @@ export function useHoverLayer() {
  * @param {object} mapRef - Ref object whose current property exposes the map instance
  * @param {object} featureCollection - A GeoJSON feature collection to fit the map bounds around
  * @param {boolean} shouldFitOnUpdate - Determines if map fits to featuresCollection if collection updates
- * @return {ViewportStateArray} Array that exposes the setter and getters for map viewport using useState hook
- */
-/**
- * @typedef {array} ViewportStateArray
- * @property {object} StateArray[0] - A Mapbox viewport object
- * @property {function} StateArray[1] - Setter for viewport state
+ * @return {function} Function to use as a Mapbox GL JS onRender callback
  */
 export function useFeatureCollectionToFitBounds(
```

Comment:
A big update here is that `useFeatureCollectionToFitBounds` no longer handles setting and getting the viewport state. This is an improvement because we don't have to alter the viewport to update the bounds of the map. We can just use Mapbox's already existing [map.fitBounds()](https://docs.mapbox.com/mapbox-gl-js/example/fitbounds/) method.

Also separates the viewport state management from this custom hook which wasn't great in the first place (but a better choice before version 7).